db/migrate/20230706141503_add_state_to_patient_sessions.rb

```diff
@@ -0,0 +1,5 @@
+class AddStateToPatientSessions < ActiveRecord::Migration[7.0]
+  def change
+    add_column :patient_sessions, :state, :string
```

Comment:
Could this be non nullable? Since we set an initial value.

```suggestion
    add_column :patient_sessions, :state, :string, null: false
```

Reply:
Good point, let me try that.

Reply:
I think we can roll forward, so I'd rather have this merged and ready to use. Then we can start integrating and iterating

Reply:
> Could this be non nullable? Since we set an initial value.

I tried this, and the fly in the ointment is that Rubocop complains, it wants us to set a default when `null: false`, which means we have do define the default in two places ... which I'm not a fan of. We then have two sources of truth.

Options:

1) Set the default column value in SQL and have two sources of truth. In fact it seems the SQL default will override the aasm default, and if they don't match aasm raises a model validation error "State is invalid". This is better than them being out of sync, but this still smells a little to me.
2) Tell Rubocop to ignore the check for a default value and not bother with it.
3) Go back to `null: true` since aasm handles this quite nicely and generally, with a few other `null:false` exceptions, we rely on in-app validations.